client/packages/system/src/Vaccination/Components/VaccineCardTable.tsx

```diff
@@ -123,7 +117,9 @@ const VaccinationCardComponent = ({
 
   const isEncounter = !!encounterId;
 
-  useStyleRowsByStatus(data?.items, isEncounter);
+  const canSkipDose = data?.items.some(item => item.canSkipDose) ?? false;
```

Comment:
I think this logic is too generic. If i have a vaccine course with 2 vaccines 1 skippable and one not skippable, then they're both show as clickable in the ui but you get an error on the non-skipable one...

<img width="1031" height="706" alt="Image" src="https://github.com/user-attachments/assets/c1bfb75e-ca11-4293-ae6c-a8e0f0346198" />

<img width="351" height="191" alt="Image" src="https://github.com/user-attachments/assets/8a7e2f80-94d0-4966-a8eb-7f1c7da30f83" />

Reply:
I think we should at least translate this error properly but probalby better to tweak the logic to avoid it in th UI...

Reply:
> I think this logic is too generic. If i have a vaccine course with 2 vaccines 1 skippable and one not skippable, then they're both show as clickable in the ui but you get an error on the non-skipable one...

Got it, yeah, I actually over-complicated it since `canSkipDose` is already on the "row", so no need to pass it in the `canClickRow` function. 

I've fixed this in the next PR though, since I modified some of the other logic around it anyway.